src/promptflow/promptflow/_sdk/entities/_run.py

```diff
@@ -51,7 +51,7 @@ class Run(YAMLTranslatableMixin):
     :type flow: Path
     :param name: Name of the run.
     :type name: Optional[str]
-    :param data: Input data for the run.
+    :param data: Input data for the run. Local path or remote uri(starts with azureml: or public URL) are supported.
```

Comment:
This is shared by local and azure. Local does not support azureml?

Reply:
Updated.